pkg/templates/machinecontroller/deployment.go

```diff
@@ -48,7 +48,7 @@ const (
 	MachineControllerAppLabelValue = "machine-controller"
 	MachineControllerImageRegistry = "docker.io"
 	MachineControllerImage         = "/kubermatic/machine-controller:"
-	MachineControllerTag           = "v1.30.0"
+	MachineControllerTag           = "v1.33.0"
```

Comment:
Can you also update the image-loader script?

https://github.com/kubermatic/kubeone/blob/7b36943d955b637a39d3852556c7c6ffb9b829dd/hack/image-loader.sh#L113

Reply:
Done 👍